src/absil/il.fs

```diff
@@ -1483,21 +1527,43 @@ type ILPropertyDefs =
     member x.AsList = let (Properties t) = x in t.Entries()
     member x.LookupByName s = let (Properties t) = x in t.[s]
 
+let convertFieldAccesFlags flags = 
+    match flags with
+    | ILMemberAccess.Assembly           -> FieldAttributes.Assembly
+    | ILMemberAccess.CompilerControlled -> failwith "Field access compiler controled."
```

Comment:
I think all mentioned of "COmpilerControlled" can be removed from the whole codebase, it's never used and removes a failure point

Reply:
Thanks I found the last build problem running the tests on my machine before pushing it